Tests/ASDisplayNodeTests.mm

```diff
@@ -480,11 +479,11 @@ - (void)checkValuesMatchSetValues:(ASDisplayNode *)node isLayerBacked:(BOOL)isLa
   XCTAssertEqualObjects(@"1 of 2", node.accessibilityValue, @"accessibilityValue broken %@", hasLoadedView);
 
   // setting the accessibilityLabel, accessibilityHint and accessibilityValue is supposed to be bridged to the attributed versions
-//  if (AS_AT_LEAST_IOS11) {
-//    XCTAssertEqualObjects(@"Ship love", node.accessibilityAttributedLabel.string, @"accessibilityAttributedLabel is broken %@", hasLoadedView);
-//    XCTAssertEqualObjects(@"Awesome things will happen", node.accessibilityAttributedHint.string, @"accessibilityAttributedHint is broken %@", hasLoadedView);
-//    XCTAssertEqualObjects(@"1 of 2", node.accessibilityAttributedValue.string, @"accessibilityAttributedValue is broken %@", hasLoadedView);
-//  }
+   if (@available(iOS 11.0, *)) {
```

Comment:
This might break some build systems on Xcode 8.3. We'll adopt this soon, but it should be done as a single diff rather than in smaller changes. Could you keep AS_AT_LEAST_IOS11 for now?

Reply:
I have no idea where this comes from tbo. I know I had that when submitting my last PR but you changed it and I thought pulling from master would keep your changes. Will change this back next week!